### ISSUE TITLE
Links to example on readme returning 404 pages

### ISSUE THREAD

**Issue by davidekete** [debug-js/debug#927]
![Screenshot 2023-03-06 at 12 31 14 PM](https://user-images.githubusercontent.com/88355936/223098645-413160f9-447c-4378-914c-0de08b5759ba.jpg)
Please review or remove links to examples in your readme. Links such as the [stdout.js](https://github.com/debug-js/debug/blob/HEAD/examples/node/stdout.js) in the screenshot above return a 404 page.

**Comment by Qix-** [comment]
Yeah the Readme needs a lot of love. The examples aren't there anymore, they were mostly pointless as the readme has a ton of examples already (`debug` is not a hard package to use).

I'll keep this open to remind myself of this when I finally get to the readme. Thanks for letting me know :)

**Comment by davidekete** [comment]
Let me know if you need any help. I can help fix the readme.

**Comment by mitant** [comment]
I've added example files matching the original README links in the following PR.  I also updated the code to pass linting and updated the README with the new lint-friendly code.

https://github.com/debug-js/debug/pull/938
